### PR TITLE
For logging, added % in place of f strings [ for performance reasons etc. #1205 ]

### PR DESCRIPTION
**Pull request by kawalpreettkaur**
1. Followed the pep8 standards
2. Removed the f strings from logging syntaxes.

[ Milestone 7.0: Discussed with @ReimarBauer  here, https://github.com/Open-MSS/MSS/issues/1205 ]

### CODE REVIEW

File: mslib/msui/mpl_qtwidget.py [hunk 1]
```diff
@@ -418,7 +418,7 @@ def mouse_move(self, event):
                 try:
                     lat, lon = self.canvas.waypoints_interactor.get_lat_lon(event)
                 except (ValueError, OverflowError) as ex:
-                    logging.error("%s", ex)
+                    logging.error("%s"%ex)
```
Comment (ReimarBauer) [inline mslib/msui/mpl_qtwidget.py]: should have blanks surrounded

File: mslib/msui/mpl_qtwidget.py [hunk 2]
```diff
@@ -740,7 +740,7 @@ def redraw_xaxis(self, lats, lons, times):
                 try:
                     self.ax.lines.remove(line)
                 except ValueError as e:
-                    logging.debug(f"Vertical line was somehow already removed:\n{e}")
+                    logging.debug("Vertical line was somehow already removed:\n%s"%e)
```
Comment (ReimarBauer) [inline mslib/msui/mpl_qtwidget.py]: blanks

File: mslib/msui/multilayers.py [hunk 3]
```diff
@@ -259,9 +259,9 @@ def get_multilayer_common_options(self, additional_layer=None):
             crs_values[0] = list(set(crs_values[0]).intersection(values))
 
         return sorted(elevation_values[0], key=lambda x: float(x.split()[0])) if len(elevation_values) > 0 else [], \
-            sorted(init_time_values[0]) if len(init_time_values) > 0 else [], \
-            sorted(valid_time_values[0]) if len(valid_time_values) > 0 else [], \
-            sorted(crs_values[0]) if len(crs_values) > 0 else []
+               sorted(init_time_values[0]) if len(init_time_values) > 0 else [], \
```
Comment (ReimarBauer) [inline mslib/msui/multilayers.py:262]: this indenting  was before pep8 verified, have you checked by flake8?

File: mslib/mscolab/utils.py [hunk 4]
```diff
@@ -64,9 +64,9 @@ def os_fs_create_dir(dir):
         try:
             _ = fs.open_fs(dir)
         except fs.errors.CreateFailed:
-            logging.error(f'Make sure that the FS url "{dir}" exists')
+            logging.error('Make sure that the FS url "{0}" exists'.format(dir))
```
Comment (ReimarBauer) [inline mslib/mscolab/utils.py]: always %s syntax (old string formatting) not .format() that has the same problem as f'strings for logging

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
Welcome Kawal Preet Kaur,
see comments, consider to verify by flake8 too.

File: mslib/msui/multilayers.py (hunk 3 above)
Comment (kawalpreettkaur) [reply mslib/msui/multilayers.py:262]: No, all the files were pep8 verified only.  For this file, I don't know how it's been modified. As far as I know, I didn't touch this file. 

Shall I still move ahead with changes at my end?

Comment (ReimarBauer) [reply mslib/msui/multilayers.py:262]: the flake8 test rules are the one I refer too. The CI checks any pull request. If it finds errors of that definition then it blocks merging. This output is from your repo which I cloned to me.

```
~/PycharmProjects/2021/kawalpreettkaur/MSS$ flake8 mslib 
mslib/msui/multilayers.py:262:16: E127 continuation line over-indented for visual indent
mslib/msui/multilayers.py:263:16: E127 continuation line over-indented for visual indent
mslib/msui/multilayers.py:264:16: E127 continuation line over-indented for visual indent
mslib/msui/mpl_qtwidget.py:421:39: E228 missing whitespace around modulo operator
mslib/msui/mpl_qtwidget.py:1051:83: E228 missing whitespace around modulo operator
mslib/msui/mss_pyui.py:256:74: E131 continuation line unaligned for hanging indent
mslib/utils/airdata.py:60:80: F509 '...' % ... has unsupported format character 'D'
mslib/utils/airdata.py:94:19: E127 continuation line over-indented for visual indent
mslib/utils/airdata.py:95:9: E122 continuation line missing indentation or outdented
mslib/utils/airdata.py:99:121: E501 line too long (134 > 120 characters)
mslib/utils/airdata.py:101:80: E126 continuation line over-indented for hanging indent
mslib/utils/airdata.py:101:121: E501 line too long (149 > 120 characters)
mslib/utils/airdata.py:102:121: E501 line too long (136 > 120 characters)
mslib/mswms/wms.py:84:5: E303 too many blank lines (2)
mslib/mswms/wms.py:113:5: E303 too many blank lines (2)
mslib/mswms/wms.py:124:5: E303 too many blank lines (2)
mslib/mswms/wms.py:131:5: E303 too many blank lines (2)
mslib/mswms/wms.py:324:52: E127 continuation line over-indented for visual indent
mslib/mswms/wms.py:382:60: E127 continuation line over-indented for visual indent
mslib/mswms/wms.py:389:121: E501 line too long (122 > 120 characters)
mslib/mswms/gallery_builder.py:489:121: E501 line too long (121 > 120 characters)
mslib/mswms/gallery_builder.py:514:12: E127 continuation line over-indented for visual indent
mslib/mswms/gallery_builder.py:515:21: E122 continuation line missing indentation or outdented
mslib/mswms/gallery_builder.py:734:31: E131 continuation line unaligned for hanging indent
mslib/mswms/mswms.py:111:72: E225 missing whitespace around operator
mslib/mswms/mswms.py:111:76: E231 missing whitespace after ','
mslib/mswms/mswms.py:114:44: E228 missing whitespace around modulo operator
mslib/mscolab/sockets_manager.py:40:50: E127 continuation line over-indented for visual indent
mslib/mscolab/sockets_manager.py:40:121: E501 line too long (122 > 120 characters)
mslib/mscolab/server.py:81:5: E303 too many blank lines (2)
mslib/mscolab/server.py:93:5: E303 too many blank lines (2)
mslib/mscolab/server.py:100:5: E303 too many blank lines (2)
```

Comment (ReimarBauer) [reply mslib/msui/multilayers.py:262]: The CI does output to:
https://github.com/Open-MSS/MSS/pull/1269/checks?check_run_id=3807857617

Comment (ReimarBauer) [reply mslib/msui/multilayers.py:262]: the CI runs on any update of your PR again. 
on clicking details see bottom of the PR I got the Url, this updates on each run too.

CIs

Comment (ReimarBauer) [reply mslib/msui/multilayers.py:262]: if files are "accidently" changed you can look on the origin in https://github.com/Open-MSS/MSS ;)

Maybe it was made by an editor configuration automatically. But it needs fixed to get merged ;)

File: mslib/mscolab/utils.py (hunk 4 above)
Comment (ReimarBauer) [reply mslib/mscolab/utils.py]: Btw. I have sent you an invitation to our slack channel.

Comment (kawalpreettkaur) [reply mslib/mscolab/utils.py]: Just joined there!

File: mslib/msui/multilayers.py (hunk 3 above)
Comment (kawalpreettkaur) [reply mslib/msui/multilayers.py:262]: Okay, looking into it.

**Comment by ReimarBauer** [comment]
see our chat for an example how to solve the merge conflict  ;)

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
there are still flake8 failures, e.g. too much blank lines. 
notneeded_changes

have a look on the flake8 output and do the needed changes.

**Comment by ReimarBauer** [comment]
@kawalpreettkaur if you want to continue here let us know. Then we can reopen this one again.

**Comment by kawalpreettkaur** [comment]
Sure thing! Is it still with the same status as in % aren't added yet?

**Comment by ReimarBauer** [comment]
Yes, I guess it is easier to create a new PR. So you don't need to care on other changes happened.

**Pull request closed without merging**